operatorhub/kubernetes/manifests/bases/tektoncd-operator.clusterserviceversion.template.yaml

```diff
@@ -0,0 +1,218 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: Developer Tools, Integration & Delivery
+    certified: "false"
+    description: Tekton Pipelines is a Kubernetes-Native CI/CD solution for building CI/CD pipelines.
+    operators.operatorframework.io/builder: operator-sdk-v1.7.2
+    repository: https://github.com/tektoncd/operator
+    support: The Tetkon Authors
+  name: tektoncd-operator.v0.0.0-0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: |
+        TektonConfig installs Tekton Pipelines, Tekton Triggers and Tekton Dashboard on a cluster. The operator installs
+        the 3 components on a cluster if an instnce of this CRD is created on your cluster with name `config`.
+        At present, the operator honors only one CustomResource instance with the name `config` to ensure only one set
+        of the three components (Pipelines, Triggers, Dashboard) exists on a cluster. An instance of this CRD will automatically
+        create instances of the other CRDs (TektonPipeline, TektonDashboard and TektonTriggers).
+      displayName: TektonConfig
+      kind: TektonConfig
+      name: tektonconfigs.operator.tekton.dev
+      version: v1alpha1
+    - description: |
+        TektonDashboard installs Tekton Dashboard on a cluster. The operator installs Tekton Dashboard on a cluster
+        if an instnce of this CRD is created on your cluster with name `dashboard`. At present, the operator honors only
+        one CustomResource instance with the name `dashboard` to ensure only one set of Tekton Dashboard exists on a cluster.
+      displayName: TektonDashboard
+      kind: TektonDashboard
+      name: tektondashboards.operator.tekton.dev
+      version: v1alpha1
+    - description: |
+        TektonPipeline installs Tekton Pipelines on a cluster. The operator installs Tekton Pipelines on a cluster
+        if an instnce of this CRD is created on your cluster with name `pipeline`. At present, the operator honors only
+        one CustomResource instance with the name `pipeline` to ensure only one set of Tekton Pipeline exists on a cluster.
+      displayName: TektonPipeline
+      kind: TektonPipeline
+      name: tektonpipelines.operator.tekton.dev
+      version: v1alpha1
+    - description: |
+        TektonTrigger installs Tekton Triggers on a cluster. The operator installs Tekton Triggers on a cluster
+        if an instnce of this CRD is created on your cluster with name `trigger`. At present, the operator honors only
+        one CustomResource instance with the name `trigger` to ensure only one set of Tekton Triggers exists on a cluster.
+      displayName: TektonTriggers
+      kind: TektonTrigger
+      name: tektontriggers.operator.tekton.dev
+      version: v1alpha1
```

Comment:
This part is dynamic isn't it ? When we add(ed) `TektonResult`, `TektonChain`, … this will need to evolve

Reply:
This file is a stub/seed from which CSV is built (not dynamic).

- Description text for CRDs
- icon
- links, keywords, etc

stuff that don't change between releases. 

But yes, TektonResult, TektonChain addition will need an entry in this file.